### PR TITLE
add papers-citing-dp

### PR DESCRIPTION
**Pull request by tuoping**
All listed in index.md.

### CODE REVIEW

**Comment by njzjz** [comment]
I can put it on the blog for a nicer view.

**Comment by tuoping** [comment]
Good idea.

File: papers-citing-dp/index.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# Papers citing DeePMD-kit[<sup>1</sup>](#refer-anchor-1)[<sup>2</sup>](#refer-anchor-2)[<sup>3</sup>](#refer-anchor-3) or DPGEN[<sup>4</sup>](#refer-anchor-4)
+
+In bibtex format
+
+- [Reviews](#reviews.md).
+- [Co-authored by E Weinan, or Zhang Linfeng, or WangHan](#Co-authored.md).
```
Comment (njzjz) [inline papers-citing-dp/index.md:6]:
```suggestion
- [Co-authored by Weinan E, or Linfeng Zhang, or Han Wang](#Co-authored.md).
```

**Comment by njzjz** [comment]
> Good idea. Although, is it convenient for everyone to push changes to blog?

Completed in
- https://deepmodeling.org/blog/papers/reviews.html
- https://deepmodeling.org/blog/papers/coauthored.html
- https://deepmodeling.org/blog/papers/others.html

They can be edited in the repository.

However, there are many duplicated papers.

**Comment by tuoping** [comment]
“However, there are many duplicated papers.”
Yes, I'm working on it.

**Comment by njzjz** [comment]
The content of the blog can be edited in 
https://github.com/deepmodeling/blog/tree/master/source/papers
https://github.com/deepmodeling/blog/blob/master/source/_data/pub.bib

**Pull request closed without merging**